server/schema/postgres.sql

```diff
@@ -37,6 +39,16 @@ CREATE TABLE IF NOT EXISTS actions (
 	updated_at timestamp NOT NULL DEFAULT now()
 );
 
+CREATE TABLE IF NOT EXISTS actiontypes (
+	id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+	organization_id uuid NOT NULL REFERENCES organizations(id) ON DELETE CASCADE ON UPDATE CASCADE,
+	name text NOT NULL,
```

Comment:
I would add a unique index over organization_id and name to prevent duplicate names and make lookups by name much faster.

Reply:
Can you show me how and how it affects the speed.

Reply:
Just use the following index definition at the end of the CREATE TABLE statement:

UNIQUE (organization_id, name)

This will ensure, that only one action type with the same name is allowed per organization. It will also make lookups by organization_id and name much faster.

Reply:
Ah okay thanks